index.bs

```diff
@@ -58,32 +58,8 @@ spec: webdriver; urlPrefix: https://w3c.github.io/webdriver/#
         text: success; url: dfn-success
 </pre>
 <pre class="link-defaults">
-spec: dom
-    type: interface
-        text: Document
-spec: html
-    type: dfn
-        text: event handler
-        text: event handler event type
-        text: in parallel
-        text: origin; for: /
-        text: queue a task
-spec: storage
-    type: enum-value
-        for: PermissionName; text: "persistent-storage"
-spec: infra
-    type: dfn
-        text: list
-        for: list; text: append
-spec: ui-events
-    type: dfn
-        text: user agent
-spec: web-bluetooth
-    type: enum-value
-        for: PermissionName; text: "bluetooth"
-spec: webidl
-    type: interface
-        text: Promise
+spec:infra; type:dfn; text:user agent
```

Comment:
I thought these were also already exported from Infra? Do we need to ping Anne to get rid of these somehow?

Reply:
These are left because we haven't set [`Assume Explicit For`](https://tabatkins.github.io/bikeshed/#metadata-assume-explicit-for) in this spec, so they're ambiguous with [report/user agent](https://www.w3.org/TR/reporting/#report-user-agent) and [URLSearchParams/list](https://url.spec.whatwg.org/#concept-urlsearchparams-list). Setting `Assume Explicit For` is a good idea, but causes ~20 errors, and I didn't want to fix them while I was chasing the Echidna problem.

Reply:
Ok cool. Thanks!